### ISSUE TITLE
Consider adding fontDisplay option to useFonts ot using `font-display: swap` by default

### ISSUE THREAD

**Issue by wojtekmaj** [WestpacGEL/GEL#772]
**Is your feature request related to a problem? Please describe.**

To avoid [FOIT](https://css-tricks.com/fout-foit-foft/), `font-display: swap` can be used so that a fallback font is used while the main font is loading.

**Describe the solution you'd like**

Non-breaking solution:

*
    ```js
    useFonts({ path: '/some/path', fontDisplay: 'swap' }); 
      // Can be 'auto', 'block', 'swap', 'fallback', 'optional'
    ```

* Leverage [`size-adjust`, `ascent-override`, `descent-override`, `line-gap-override`](https://web.dev/css-size-adjust/) to prevent cumulative layout shift (this can be done independently)

Breaking solution:

* Add `font-display: swap` by default. 
* Leverage [`size-adjust`, `ascent-override`, `descent-override`, `line-gap-override`](https://web.dev/css-size-adjust/) to prevent cumulative layout shift (this can be done independently)

**Describe alternatives you've considered**

```js
const fonts = useFonts({ path: '/static/fonts/' });
fonts[''].forEach((font) => { font['@font-face'].fontDisplay = 'swap'; });
```

🤢

**Comment by dominikwilkowski** [comment]
I totally agree with your sentiment and we had a long internal discussion about this. Adding an option would be a good idea actually. Will see when I get time next.

Can I ask where you're using this lib? Would be good to understand more about your use case

**Comment by wojtekmaj** [comment]
@dominikwilkowski Not sure if I can share that publicly, so I dropped you a DM on Twitter.

**Comment by jonnystening** [comment]
@wojtekmaj feel free to contact us directly if you need anything gel@westpac.com.au or via Teams

**Comment by wojtekmaj** [comment]
Sorry, Teams? How do I join? I see only Slack in your footer, but it doesn't seem to be public. Would love to chat with you guys from time to time.

**Comment by dominikwilkowski** [comment]
This was merged btw: #773
Closing this. I'm sure it will be published soon
